### DISCUSSION TITLE
Trying to use lexicons/app/bsky/feed/getRepostedBy but hitting the limit

### DISCUSSION THREAD

**Discussion by ryandhasgit** [bluesky-social/atproto#1482] [How to do it?] [answered]
For my feed, I have created a way to opt in and out of appearing, and it involves reposting a single post on an account. When the indexing logic of the feed sees those reposts, it knows who belongs on it.

However, the issue I'm running into is that this API is capped at 100 and there is no way to get more, even to paginate through.  Obviously there are other ways to register a person for my feed, but I quite like this one. Is there any way around this limit?

https://github.com/bluesky-social/atproto/blob/main/lexicons/app/bsky/feed/getRepostedBy.json

**Comment by dholms** [comment, marked as answer]
You can paginate by providing the cursor in the response in the next request's params

**Reply by ryandhasgit** [reply]
Perfect, thank you
